extra/release_tool.py

```diff
@@ -1289,9 +1289,8 @@ def tag_or_followed_branch(repo_git):
             return
 
         executed = query_execute_list([
-            ["docker", "run", "-d", "--name", "release_tool_gui_licenses", gui_tag],
-            ["docker", "cp", "release_tool_gui_licenses:/var/www/mender-gui/dist/disclaimer.txt", "gui-licenses.txt"],
-            ["docker", "rm", "-f", "release_tool_gui_licenses"],
+            ["docker", "run", "--rm", "--entrypoint", "/bin/sh", "-v", os.getcwd() + ":/extract", gui_tag,
+            "-c", "cp disclaimer.txt /extract/gui-licenses.txt"]
```

Comment:
This solution has the root-problem, that the file is owned by root after this operation.

Reply:
I reverted back to the original for now, as changing the file location seemed a simpler change than adding `chown`.

Reply:
:+1: